modules/network-observability-operator.adoc

```diff
@@ -0,0 +1,9 @@
+// Module included in the following assemblies:
+//
+// network_observability/network-observability-overview.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="network-observability-operator_{context}"]
+= Network Observability Operator
+
+The Network Observability Operator provides the `FlowCollector` API custom resource. A `FlowCollector` instance is a cluster-scoped resource that enables configuration of network flow collection. This instance deploys pods and services that form a monitoring pipeline. The pipeline collects and enriches network flows with Kubernetes metadata before storing them in Loki or generating Prometheus metrics. The `eBPF` agent, which is deployed as a `daemonset`, creates the network flows.
```

Comment:
We should mention the eBPF agent first since it generates the flows ?
Then the pipeline enrich those and save to loki and or prometheus
And finally the plugin consume the data